requirements.txt

```diff
@@ -32,4 +67,19 @@ ruamel.yaml.clib==0.2.0 \
 six==1.16.0 \
     --hash=sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926 \
     --hash=sha256:8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254
-    # via operator-manifest (setup.py)
+    # via
+    #   jsonschema
+    #   operator-manifest (setup.py)
+typing-extensions==3.10.0.0 \
+    --hash=sha256:0ac0f89795dd19de6b97debb0c6af1c70987fd80a2d62d1958f7e56fcc31b497 \
+    --hash=sha256:50b6f157849174217d0656f99dc82fe932884fb250826c18350e159ec6cdf342 \
+    --hash=sha256:779383f6086d90c99ae41cf0ff39aac8a7937a9283ce0a414e5dd782f4c94a84
+    # via importlib-metadata
+zipp==3.5.0 \
+    --hash=sha256:957cfda87797e389580cb8b9e3870841ca991e2125350677b2ca83a0e99390a3 \
+    --hash=sha256:f5812b1e007e48cff63449a5e9f4e7ebea716b4111f9c4f9a645f91d579bf0c4
+    # via importlib-metadata
+
+# WARNING: The following packages were not pinned, but pip requires them to be
```

Comment:
:eyes: are we ok with this?

Reply:
according to pip-compile help, distribute,pip, and setuptools are considered unsafe to pin - but there is the option of --alow-unsafe which will be the default behavior anyway. So I would just use that flag.

Reply:
https://github.com/containerbuildsystem/operator-manifest/blob/master/Dockerfile  can you update dockerfile then?

Reply:
nevermind, I was informed that it applies only to pip-compile